readthedocs/core/tasks.py

```diff
@@ -56,3 +58,31 @@ def clear_persistent_messages():
         expires__lt=timezone.now(),
     )
     expired_messages.delete()
+
+
+@app.task(queue="web")
+def cleanup_pidbox_keys():
+    """
+    Remove "pidbox" objects from Redis.
+
+    Celery creates some "pidbox" objects with TTL=-1,
+    producing a OOM on our Redis instance.
+
+    This task is executed periodically to remove "pdibox" objects with an
+    idletime bigger than 5 minutes from Redis and free some RAM.
+
+    https://github.com/celery/celery/issues/6089
+    https://github.com/readthedocs/readthedocs-ops/issues/1260
+
+    """
+    total_memory = 0
+    keys = app.backend.client.keys("*reply.celery.pidbox*")
```

Comment:
`app.backend.client` is not working anymore since we are ignoring results and backed is instantiated as `DisabledBackend` now. We have to use `from django.core.cache import cache`, I think.

Reply:
Yea, just need to make sure we're connecting to the right redis database.